content/articles/making-predictions-using-monte-carlo-simulation/index.md

```diff
@@ -0,0 +1,299 @@
+### Table of Contents
+
+-[Introduction](#introduction)
+
+-[What is Monte Carlo Simulation?](#what-is-monte-carlo-simulation)
+
+-[Unique attributes from other techniques](#unique-attributes-from-other-techniques)
+
+-[Understanding Monte Carlo basics](#understanding-monte-carlo-basics)
+
+-[Examples](#examples)
+
+-[Real Time Application](real-time-application)
+
+-[How accurate](#how-accurate)
+
+-[Why prefer Monte Carlo?](why-prefer-monte-carlo-?)
+
+-[Why financial advisors say NO to this technique](why-most-people-say-no-to-this-technique)
+
+-[Conclusion](conclusion)
+
+### Introduction
+
+With the growing technology especially in the field of AI(Artificial Intelligence), the knowledge of different methods of training a model is essential to programmers.
+
+These techniques help in automation of decision making process and help predict the probability of different outcomes when random variables are intercepted.
+
+This article will simplify monte carlo method operations and help programmers to clearly understand what this technique entail.
+
+### What is Monte Carlo Simulation?
+It is simply a model that is used to outline the probability of various outcomes of decisions made. The simulations help to describe and assess the expected risk impact to eliminate uncertainity for better decisions.
+
+This is an approach technique rather than an exact technique.
+
+**Pseudo-random number generators** are used when implementing using computers.
+
+It is frequently utilized in variety of industries, including engineering, finance, business, project planning, and many more.
+
+### Unique attributes from other techniques
```

Comment:
if these are the attributes, how are they supposed to be numbered? 🤔

Reply:
Okay, i think they are now correctly numbered